commons/ava_services/gecko_service_initializer_core.go

```diff
@@ -72,7 +85,37 @@ func (g GeckoServiceInitializerCore) GetUsedPorts() map[int]bool {
 	}
 }
 
-func (g GeckoServiceInitializerCore) GetStartCommand(publicIpAddr string, dependencies []services.Service) []string {
+
+func (g GeckoServiceInitializerCore) GetFilepathsToMount() map[string]bool {
+	if g.stakingTlsEnabled {
+		return map[string]bool{
+			stakingTlsCertPath: true,
+			stakingTlsKeyPath: true,
+		}
+	}
+	return make(map[string]bool)
+}
+
+func (g GeckoServiceInitializerCore) InitializeMountedFiles(osFiles map[string]*os.File, dependencies []services.Service) (err error) {
+	certFilePointer := osFiles[stakingTlsCertPath]
+	keyFilePointer := osFiles[stakingTlsKeyPath]
+	if len(dependencies) == 0 {
```

Comment:
There should be a big fat TODO here that there's a bug where `numBootNodes` > 1 will cause nodes with duplicate IDs because they're hardcoded

Reply:
not duplicate ids, but rather we don't have their IDs. i'll add the TODO

Reply:
done